plugins/Live/API.php

```diff
@@ -146,6 +146,14 @@ public function getLastVisitsForVisitor($visitorId, $idSite, $filter_limit = 10,
     }
 
     /**
+     * Proxy for @see getLastVisitsDetails
+     */
+    public function getLastVisitsDetailsWidget($idSite, $period = false, $date = false, $segment = false, $countVisitorsToFetch = false, $minTimestamp = false, $flat = false, $doNotFetchActions = false)
+    {
+        return $this->getLastVisitsDetails($idSite, $period, $date, $segment, $countVisitorsToFetch, $minTimestamp, $flat, $doNotFetchActions);
+    }
```

Comment:
You could possibly overwrite the `fetch()` method in the Report then this Magic won't need a new method anymore. Let me know how it goes, happy to help.